gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -616,8 +618,13 @@ private ManagedChannel createSingleChannel() throws IOException {
     boolean useDirectPathXds = false;
     if (canUseDirectPath()) {
       CallCredentials callCreds = MoreCallCredentials.from(credentials);
+      // altsCallCredentials may be null and will be correctly ignored by
+      // GoogleDefaultChannelCredentials.
```

Comment:
can you add something like may be null when X... I think from your comment below that it's for cloudpath scenarios?

Reply:
Sorry I didn't explain very clearly. This may be null if we don't enable the bound token for DirectPath. And gRPC will use the original credentials for everything (DirectPath or CloudPath). Otherwise, gRPC will use it for DirectPath connections and the original credentials for CloudPath fallback.

I added a sentence for this.